python/tvm/script/printer/doc.py

```diff
@@ -100,6 +100,31 @@ def __iter__(self):
         raise RuntimeError(f"{self.__class__} cannot be used as iterable.")
 
 
+class StmtDoc(Doc):
+    """Base class of statement doc"""
+
+    @property
+    def comment(self) -> Optional[str]:
+        # It has to call the dunder method to avoid infinite recursion
+        # pylint: disable=unnecessary-dunder-call
+        return self.__getattr__("comment")
+        # pylint: enable=unnecessary-dunder-call
```

Comment:
nit if you only want to disable specific lint on a single line

```suggestion
        return self.__getattr__("comment")  # pylint: disable=unnecessary-dunder-call
```